src/jsi/jsi.cpp

```diff
@@ -347,7 +368,11 @@ JSError::JSError(Runtime& rt, Value&& value) {
 JSError::JSError(Runtime& rt, std::string msg) : message_(std::move(msg)) {
   try {
     setValue(
-        rt, rt.global().getPropertyAsFunction(rt, "Error").call(rt, message_));
+        rt,
+        callGlobalFunction(rt, "Error", String::createFromUtf8(rt, message_)));
+  } catch (const std::exception& ex) {
+    message_ = std::string(ex.what()) + " (while raising " + message_ + ")";
+    setValue(rt, String::createFromUtf8(rt, message_));
```

Comment:
>setValue [](start = 4, length = 8)

What if this one throws? Or we only want to catch "callGlobalFunction"? If so should we wrap it to "callGlobalFunction" only?